tools/generator-bot-template-generator/app/index.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+const path = require('path');
+const Generator = require('yeoman-generator');
+const askName = require('inquirer-npm-name');
+const _ = require('lodash');
+const extend = require('deep-extend');
+const mkdirp = require('mkdirp');
+
+function makeGeneratorName(name) {
+  name = _.kebabCase(name);
```

Comment:
lol kebab case

Reply:
Any change needed here or just amused? Its a simple little function. Could always write it ourselves but we already have the lodash dependency so may as well use it.